com.github.giacomogroppi.writernote-qt.json

```diff
@@ -0,0 +1,79 @@
+{
+  "app-id": "com.github.giacomogroppi.writernote-qt",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.2",
+  "sdk": "org.kde.Sdk",
+  "command": "writernote",
+  "rename-appdata-file": "com.github.giacomogroppi.writernote-qt.appdata.xml",
```

Comment:
I don't think this is necessary as it already has this name.
`Renaming com.github.giacomogroppi.writernote-qt.appdata.xml to share/appdata/com.github.giacomogroppi.writernote-qt.appdata.xml`